var/spack/repos/builtin/packages/py-neurom/package.py

```diff
@@ -31,6 +32,7 @@ class PyNeurom(PythonPackage):
     depends_on('py-matplotlib@3.2.1:', type='run')
     depends_on('py-scipy@1.2.0:', type='run')
     depends_on('py-plotly@3.6.0:', type='run', when='+plotly')
+    depends_on('py-morphio@3.1.1:', type='run', when='@2.0:2.999')
```

Comment:
There should be also a dependency for pandas. Maybe you can use the same changes for neurom made in the PR https://github.com/BlueBrain/spack/pull/1099/files to avoid merge conflicts later?